File: src/com/atl/atlmovil/entidades/Pedido.java
package com.atl.atlmovil.entidades;

import java.util.Date;
import java.util.List;

public class Pedido {
private long id;// id interno
private long codigoPedido;
private long codigoVisita; // con esto ya se tiene el codigo de cliente
private long codigoFormaPago;
private long codigoEmpresaCarga;
private Date fechaIngresoPedido;
private Double importePedido;
private Double lineaReservadaPedido;
private Boolean aceptaRetencionPedido;
private Boolean estaRetenidoPedido;
private String estadoPedido; // A -> Activo, N-> Anulado, R -> Registrado, 
private Boolean estaSincronizado; //1 -> sincronizado, 0-> pendiente de sincronizacion
private String direccionDeEnvio;
private String empresaTransporte;
private String instruccionesEspeciales;
private String strfechaIngresoPedido;
private String codigoMovil;

private List<DetallePedido> detalles;


public long getId() {
	return id;
}
public void setId(long id) {
	this.id = id;
}
public long getCodigoPedido() {
	return codigoPedido;
}
public void setCodigoPedido(long codigoPedido) {
	this.codigoPedido = codigoPedido;
}
public long getCodigoVisita() {
	return codigoVisita;
}
public void setCodigoVisita(long codigoVisita) {
	this.codigoVisita = codigoVisita;
}
public long getCodigoFormaPago() {
	return codigoFormaPago;
}
public void setCodigoFormaPago(long codigoFormaPago) {
	this.codigoFormaPago = codigoFormaPago;
}
public Date getFechaIngresoPedido() {
	return fechaIngresoPedido;
}
public void setFechaIngresoPedido(Date fechaIngresoPedido) {
	this.fechaIngresoPedido = fechaIngresoPedido;
}
public Double getImportePedido() {
	return importePedido;
}
public void setImportePedido(Double importePedido) {
	this.importePedido = importePedido;
}
public Double getLineaReservadaPedido() {
	return lineaReservadaPedido;
}
public void setLineaReservadaPedido(Double lineaReservadaPedido) {
	this.lineaReservadaPedido = lineaReservadaPedido;
}
public Boolean getAceptaRetencionPedido() {
	return aceptaRetencionPedido;
}
public void setAceptaRetencionPedido(Boolean aceptaRetencionPedido) {
	this.aceptaRetencionPedido = aceptaRetencionPedido;
}
public Boolean getEstaRetenidoPedido() {
	return estaRetenidoPedido;
}
public void setEstaRetenidoPedido(Boolean estaRetenidoPedido) {
	this.estaRetenidoPedido = estaRetenidoPedido;
}
public String getEstadoPedido() {
	return estadoPedido;
}
public void setEstadoPedido(String estadoPedido) {
	this.estadoPedido = estadoPedido;
}
public Boolean getEstaSincronizado() {
	return estaSincronizado;
}
public void setEstaSincronizado(Boolean estaSincronizado) {
	this.estaSincronizado = estaSincronizado;
}
public String getDireccionDeEnvio() {
	return direccionDeEnvio;
}
public void setDireccionDeEnvio(String direccionDeEnvio) {
	this.direccionDeEnvio = direccionDeEnvio;
}
public String getEmpresaTransporte() {
	return empresaTransporte;
}
public void setEmpresaTransporte(String empresaTransporte) {
	this.empresaTransporte = empresaTransporte;
}
public String getInstruccionesEspeciales() {
	return instruccionesEspeciales;
}
public void setInstruccionesEspeciales(String instruccionesEspeciales) {
	this.instruccionesEspeciales = instruccionesEspeciales;
}
public long getCodigoEmpresaCarga() {
	return codigoEmpresaCarga;
}
public void setCodigoEmpresaCarga(long codigoEmpresaCarga) {
	this.codigoEmpresaCarga = codigoEmpresaCarga;
}
public List<DetallePedido> getDetalles() {
	return detalles;
}
public void setDetalles(List<DetallePedido> detalles) {
	this.detalles = detalles;
}
public String getStrfechaIngresoPedido() {
	return strfechaIngresoPedido;
}
public void setStrfechaIngresoPedido(String strfechaIngresoPedido) {
	this.strfechaIngresoPedido = strfechaIngresoPedido;
}
public String getCodigoMovil() {
	return codigoMovil;
}
public void setCodigoMovil(String codigoMovil) {
	this.codigoMovil = codigoMovil;
}

}

File: net/ATL.Servicio/ATL.Entidad/Pedido.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ATL.Entidad
{
    public class Pedido
    {
        /*
         11-14 10:24:04.747: I/test pedido(227): {"aceptaRetencionPedido":false,"strfechaIngresoPedido":"2013-20-14 10:20",
         * "lineaReservadaPedido":0.0,"instruccionesEspeciales":"tyhf",
         * "importePedido":0.0,"detalles":[{"tallas":[{"idPedido":1,"codigoProducto":1,"cantidad":4,"numeroTalla":39},
         * {"idPedido":1,"codigoProducto":1,"cantidad":4,"numeroTalla":40}],"idPedido":1,"precioUnitario":0.0,"codigoProducto":1}],
         * "direccionDeEnvio":"Av. De la Conquista 789, Santiago de Surco","estaRetenidoPedido":false,
         * "estaSincronizado":false,"estadoPedido":"Ingresado","fechaIngresoPedido":"Nov 14, 2013 10:20:00 AM","id":1,
         * "codigoVisita":16,"codigoPedido":0,"codigoFormaPago":2,"codigoEmpresaCarga":2}
         */

        public long id { get; set; } //temporal
        public long codigoPedido { get; set; }
        public long codigoVisita { get; set; }
        public long codigoFormaPago { get; set; }
        public long codigoEmpresaCarga { get; set; }
        public Double importePedido { get; set; }
        public Double lineaReservadaPedido { get; set; }
        public Boolean aceptaRetencionPedido { get; set; }
        public Boolean estaRetenidoPedido { get; set; }
        public String estadoPedido { get; set; } // A -> Activo, N-> Anulado, R -> Registrado, 
        public Boolean estaSincronizado { get; set; } //1 -> sincronizado, 0-> pendiente de sincronizacion
        public String direccionDeEnvio { get; set; }
        public String empresaTransporte { get; set; }
        public String instruccionesEspeciales { get; set; }
        public String strfechaIngresoPedido { get; set; }

        public List<DetallePedido> detalles;



    }
}
